src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/TestFiles/DefaultRazorSemanticTokenInfoServiceTest/GetSemanticTokensEdits_CSharp_RazorIfNotReady.semanticedit.txt

```diff
@@ -0,0 +1 @@
+Delta
```

Comment:
?

Reply:
This ends up meaning that there was no change. It's just how we represented things to serialize them like this.